lib/nexmo_developer/app/assets/javascripts/digitalData.js

```diff
@@ -1,11 +1,13 @@
+const PRIMARY_CATEGORY = document.getElementById('adobe_launch_primary_category').value
+
 window.digitalData = {
   page: {
     pageInfo: {
       pageName: '',
-      functionDept: 'biz:api:adp',
-      primaryCategory: 'biz:api:adp:developer',
-      siteIdentifier: 'api:adp:developer',
-      lob: 'biz',
+      functionDept: '',
+      primaryCategory: PRIMARY_CATEGORY,
+      siteIdentifier: '',
+      lob: '',
```

Comment:
Do we need environment variables for functionDept, siteIdentifier & lob?

Reply:
Do you want me to remove them ?

Reply:
Do you want me to set them as environment variables ?